graphql-schemas/schema.yml

```diff
@@ -1977,13 +1977,45 @@ confs:
 
 - name: NamespaceTerraformResourceALBRules_v1
   fields:
-  - { name: condition, type: NamespaceTerraformResourceALBConditon_v1, isRequired: true}
+  - { name: condition, type: NamespaceTerraformResourceALBCondition_v1, isRequired: true, isList: true, isInterface: true}
   - { name: action, type: NamespaceTerraformResourceALBAction_v1, isRequired: true, isInterface: true}
 
-- name: NamespaceTerraformResourceALBConditon_v1
+- name: NamespaceTerraformResourceALBCondition_v1
+  isInterface: true
+  interfaceResolve:
+    strategy: fieldMap
+    field: type
+    fieldMap:
+      host-header: NamespaceTerraformResourceALBConditionHostHeader_v1
+      http-request-method: NamespaceTerraformResourceALBConditionHTTPRequestMethod_v1
+      path-pattern: NamespaceTerraformResourceALBConditionPathPattern_v1
+      source-ip: NamespaceTerraformResourceALBConditionSourceIP_v1
+  fields:
+  - { name: type, type: string, isRequired: true }
```

Comment:
I'm sure there's a good reason for it, but I was just curious if `type` is redundant here?

```yaml
  - type: source-ip
    source_ip:
    - 10.11.12.13
```

I'm guessing there are difficulties with moving `source_ip` to being a field on `NamespaceTerraformResourceALBCondition_v1`?

It's not the biggest deal, just more curious than anything.